x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/document_creation_buttons.tsx

```diff
@@ -187,6 +187,16 @@ export const DocumentCreationButtons: React.FC<Props> = ({
           <EuiCard
             hasBorder
             layout="horizontal"
+            betaBadgeProps={{
+              label: 'Beta',
```

Comment:
We need to `i18n` this too
```suggestion
              label: i18n.translate(
                'xpack.enterpriseSearch.appSearch.documentCreation.buttons.elasticsearchBetaBadge',
                {
                  defaultMessage: 'Beta',
                }
              ),
```

Reply:
Might want to change that locally and then push in case the whitespace doesn't translate and the linter fails. You can try just committing that #yolo